app/models/location_polygon.rb

```diff
@@ -13,6 +13,8 @@ def self.with_name(location)
   end
 
   def self.include?(location)
+    return super unless location.is_a? String
+
```

Comment:
So yeah, that is a bit hacky. Would be nice if we could rename this method so that we don't override basic Module interface...

Reply:
Yes I was thinking about that too. I wanted to separate the PRs. But good point

Reply:
@BroiSatse there are a few cases of included where I'm not sure if we're passing a polygon or not. It needs a bit of investigation. I'll raise a separate PR for it